client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js

```diff
@@ -418,17 +433,31 @@ export default class CreateOrUpdateForm extends React.Component {
                 }
             }
 
+            if (longitude.value.includes('°')) {
+                longitude.value = longitude.value.replaceAll(/\s/g, '');
+                if (!validator.matches(longitude.value, longitudeDegRegex)) {
+                    longitude.isValid = false;
+                    longitude.message = 'Longitude must be valid';
+                    formIsValid = false;
+                } else {
+                    const lonValues = longitude.value.split(/\°|\'|\"/g);
+                    const lonDegree = parseFloat(lonValues[0]);
+                    const lonMin = parseFloat(lonValues[1]);
+                    const lonSec = parseFloat(lonValues[2]);
+
+                    //decimal = degrees + (minutes/60) + (seconds/3600)
+                    let lonDecimal = Math.abs(lonDegree) + (lonMin/60) + (lonSec/3600);
+                    if ((longitude.value.includes('W'))||(longitude.value.includes('w'))||(lonDegree<0)) {
+                        lonDecimal *= -1;
+                    }
+                    longitude.value = lonDecimal.toString();
+                }
+            }
```

Comment:
could be a function with lat or long passed in to avoid dupe code

Reply:
Done